project/plugins.sbt

```diff
@@ -1,21 +1,7 @@
-import sbt._
+resolvers += "Era7 maven releases" at "https://s3-eu-west-1.amazonaws.com/releases.era7.com"
```

Comment:
this is not needed since you are filtering codacy resolvers in build.sbt

Reply:
it fails